lib/Doctrine/DBAL/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -84,14 +84,14 @@ public function execute(InputInterface $input, OutputInterface $output)
         } else {
             $noInteraction = $input->getOption('no-interaction') ? true : false;
             if ($noInteraction === true) {
-                $version->execute($direction, $input->getOption('dry-run') ? true : false);
+                $confirmation = true;
```

Comment:
Maybe changing the name of the variable to $execute make it a bit more understandable ?